hw/ip/rbox/rbox.core

```diff
@@ -0,0 +1,68 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:rbox:1.0"
+description: "rbox"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:constants:top_pkg
+      - lowrisc:prim:all
+      - lowrisc:ip:tlul
+    files:
+      - rtl/rbox_reg_pkg.sv
+      - rtl/rbox_reg_top.sv
+      - rtl/rbox_autoblock.sv
+      - rtl/rbox_comboact.sv
+      - rtl/rbox_combotrg.sv
+      - rtl/rbox_intr.sv
+      - rtl/rbox_inv.sv
+      - rtl/rbox_pin.sv
+      - rtl/rbox_keyintr.sv
+      - rtl/rbox_timerfsm.sv
+      - rtl/rbox_keyfsm.sv
+      - rtl/rbox_combofsm.sv
+      - rtl/rbox_combo.sv
+      - rtl/rbox.sv
+    file_type: systemVerilogSource
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+    file_type: waiver
+
+  files_veriblelint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+
+parameters:
+  SYNTHESIS:
+    datatype: bool
+    paramtype: vlogdefine
+
+targets:
+  default: &default_target
+    filesets:
+      - tool_ascentlint  ? (files_ascentlint_waiver)
+      - files_rtl
+    toplevel: rbox
+
+
+  lint:
+    <<: *default_target
+    default_tool: verilator
+    parameters:
+      - SYNTHESIS=true
+    tools:
```

Comment:
@msfschaffner, I guess this part was cleaned up, right?

Reply:
yes that's right. please replace this `lint:` target with the following snippet:
```
  lint:
    <<: *default_target
    default_tool: verilator
    parameters:
      - SYNTHESIS=true
    tools:
      verilator:
        mode: lint-only
        verilator_options:
          - "-Wall"
```

Reply:
Fixed.